draft-ietf-tls-tls13.md

```diff
@@ -1937,12 +1943,24 @@ random
   This structure is generated by the server and MUST be
   generated independently of the ClientHello.random.
 
+legacy_session_id_echo
+: The contents of the client's legacy_session_id field. Note that
+  this field is echoed even if the client's value corresponded to
+  a cached pre-TLS 1.3 session which the server has chosen not
+  to resume. A client which receives a legacy_session_id field
```

Comment:
We should consider saying something about how the TLS 1.3-aware client must do the version check before the resumption check, in case some existing TLS 1.2 code does a resumption/sesion-id check early in processing and would be confused about what protocol is in use.

Reply:
Done.